scripts/Vulkan.py

```diff
@@ -12,10 +12,12 @@
 VULKAN_SDK = os.environ.get('VULKAN_SDK')
 VULKAN_SDK_INSTALLER_URL = 'https://sdk.lunarg.com/sdk/download/1.2.170.0/windows/vulkan_sdk.exe'
 HAZEL_VULKAN_VERSION = '1.2.170.0'
+VULKAN_SDK_EXE_DIR = 'Ghost/vendor/VulkanSDK'
```

Comment:
This should be changed to ``Hazel/vendor/VulkanSDK``

Reply:
Yeah, my bad.

Reply:
Done